### ISSUE TITLE
Can't switch cameras

### ISSUE THREAD

**Issue by odahcam** [werthdavid/ngx-zxing#16]
Sometimes (just sometimes, not always), when I try to change my camera source (with that select input) I got this error and cameras doesn't change:

```
ERROR Error: Uncaught (in promise): [object NavigatorUserMediaError]
    at resolvePromise (zone.js:824)
    at resolvePromise (zone.js:795)
    at eval (zone.js:873)
    at ZoneDelegate.invokeTask (zone.js:425)
    at Object.onInvokeTask (ng_zone.js:575)
    at ZoneDelegate.invokeTask (zone.js:424)
    at Zone.runTask (zone.js:192)
    at drainMicroTaskQueue (zone.js:602)
    at <anonymous>
``` 

![image](https://user-images.githubusercontent.com/3942006/35328597-06ebbebc-00e4-11e8-85f3-6d3f3d4a26ba.png)

**Comment by odahcam** [comment]
A method `setDevice` should be created to handle the device change and prevent strange behaviors.

**Comment by odahcam** [comment]
Related: https://developers.google.com/web/updates/2017/06/play-request-was-interrupted

**Comment by odahcam** [comment]
Fixed in #28
